golang/cosmos/Makefile

```diff
@@ -73,17 +73,18 @@ compile-gyp:
 	node-gyp configure build $(GYP_DEBUG) || { status=$$?; rm -f binding.gyp; exit $$status; }
 	rm -f binding.gyp
 
-compile-libdaemon: go-mod-cache
+compile-libdaemon: go.sum
 	go build -v $(MOD_READONLY) $(SHARED_BUILD_FLAGS) -buildmode=c-shared \
 		-o build/libagcosmosdaemon.so ./cmd/libdaemon/main.go
 
-go-mod-cache: go.sum
-	@echo "--> Download go modules to local cache"
-	@go mod download
-
 go.sum: go.mod
-	@echo "--> Ensure dependencies have not been modified"
+	@echo "--> Ensure dependencies have not been modified unless suppressed by SKIP_MOD_VERIFY ..."
```

Comment:
Change message to "Ensure dependencies have not been modified (suppress with SKIP_MOD_VERIFY)" and move it inside the `ifndef`, so that if you do suppress it, nothing is printed.

Reply:
I'm intentionally printing it in both cases, to help avoid potential surprise like "_why didn't this CI run verify modules?_". Do you feel strongly about not doing so?

Reply:
With intermediate layers of scripts invoking `make` (e.g. `./bin/agd`), it's not always obvious when you're reading the log what the command-line arguments were. I think logging the actions is the most straightforward approach, since it's impractical to accurately log actions not taken, and why.